components/features/FeatureList.js

```diff
@@ -1,50 +1,56 @@
 export const features = [
```

Comment:
@anshgoyalevil 

Do we need that file when we have this content inside `locales`? Except that question, everything is good :)

Reply:
Yes. Actually we are using the ids of those features to create unique translation keys to fetch those from the locales

Reply:
Like here for example
```
{t(`features.${feature.id}.name`)}
```

Reply:
So maybe put another texts as values to keys, not whole content, ok?

Reply:
Done with the changes 🚀